api/src/main/java/org/opennms/integration/api/v1/config/poller/Monitor.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2011-2014 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.integration.api.v1.config.poller;
+
+import java.util.List;
+
+/**
+ * Monitor for a service
+ */
+public interface Monitor{
+    
+    /**
+     * Service name
+     */
+    String getService();
+
+    /**
+     * Fully-qualified Java class name used to monitor/poll the service. The class must implement the
+     * org.opennms.netmgt.poller.monitors.ServiceMonitor interface.
```

Comment:
This is not true for the poller exposed as a plugin. They must implement `org.opennms.integration.api.v1.pollers.ServicePoller`.